scripts/zones/Norg/Zone.lua

```diff
@@ -4,54 +4,49 @@
 --
 -----------------------------------
 local ID = require("scripts/zones/Norg/IDs")
+require("scripts/globals/chocobo")
 require("scripts/globals/conquest")
 require("scripts/globals/keyitems")
 require("scripts/globals/missions")
-require("scripts/globals/chocobo")
+require("scripts/globals/npc_util")
 -----------------------------------
 
 function onInitialize(zone)
     dsp.chocobo.initZone(zone)
-end;
+end
 
 function onConquestUpdate(zone, updatetype)
     dsp.conq.onConquestUpdate(zone, updatetype)
-end;
-
-function onZoneIn(player,prevZone)
+end
 
-    local cs = -1;
+function onZoneIn(player, prevZone)
+    local cs = -1
 
-    if (player:getXPos() == 0 and player:getYPos() == 0 and player:getZPos() == 0) then
-        player:setPos(-19.238,-2.163,-63.964,187);
+    if player:getXPos() == 0 and player:getYPos() == 0 and player:getZPos() == 0 then
+        player:setPos(-19.238,-2.163,-63.964,187)
     end
-    if (player:getCurrentMission(ZILART) == dsp.mission.id.zilart.THE_NEW_FRONTIER) then
-        cs = 1;
-    elseif (player:getCurrentMission(ZILART) == dsp.mission.id.zilart.AWAKENING and player:getVar("ZilartStatus") == 0 or player:getVar("ZilartStatus") == 2) then
-        cs = 176;
+    if player:getCurrentMission(ZILART) == dsp.mission.id.zilart.THE_NEW_FRONTIER then
+        cs = 1
+    elseif player:getCurrentMission(ZILART) == dsp.mission.id.zilart.AWAKENING and player:getVar("Awakening_Norg") == 1 then
```

Comment:
why are you changing the variable name and value for a long standing working mission.

Reply:
seemed much simpler check to check 2 cs's in any order

Reply:
it worked just fine with ZilartStatus and convention has always been expansion+status for the variable name on missions. That isn't changing without a serious discussion.